tetris/piece.go

```diff
@@ -4,6 +4,28 @@ import (
 	"math/rand"
 )
 
+const (
+    ColorReset  = "\033[0m"
+    ColorRed    = "\033[31m"
+    ColorGreen  = "\033[32m"
+    ColorYellow = "\033[33m"
+    ColorBlue   = "\033[34m"
+    ColorPurple = "\033[35m"
+    ColorCyan   = "\033[36m"
+    ColorWhite  = "\033[37m"
+)
+
+
+var pieceColors = []string{
```

Comment:
You need to create an enum like type:

type Color string

const () // this is what you have now. 

then:

var pieceColors = []Color{...}